extensions/mssql/src/features.ts

```diff
@@ -76,7 +76,7 @@ export class AccountFeature implements StaticFeature {
 			window.showErrorMessage(unauthorizedMessage);
 			return undefined;
 		}
-		const securityToken = await azdata.accounts.getAccountSecurityToken(account, tenant, azdata.AzureResource.AzureKeyVault);
+		const securityToken = await azdata.accounts.getAccountSecurityToken(account, tenant.id, azdata.AzureResource.AzureKeyVault);
```

Comment:
I'm surprised this wasn't prevented by parameter typing.  I expected that the compiler would catch this.

Reply:
Tenant is an any type so that's why it wasn't caught. 

This is currently because the account type is a generic azdata.Account which doesn't have the concept of tenants (those are a specific implementation for Azure accounts). 

That being said we should most definitely also be fixing this to have proper typing for Azure accounts - stuff like this is why I keep pushing people to never use any unless you absolutely need to and then to create types whenever you can if you have specific implementations.